packages/strapi-plugin-upload/admin/src/components/SelectAll/index.js

```diff
@@ -1,13 +1,26 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import { Checkbox } from '@buffetjs/core';
 import Wrapper from './Wrapper';
 
-const SelectAll = () => {
+const SelectAll = ({ checked, onChange, someChecked }) => {
```

Comment:
I don't understand the `someChecked` prop.

Reply:
The `someChecked` prop is used for the checkbox to display a dash when some items are selected and not all of them. The buffetjs `Checkbox` compo uses a `someChecked` prop too.

<img width="132" alt="Capture d’écran 2020-03-10 à 16 39 58" src="https://user-images.githubusercontent.com/3142866/76330199-da739300-62ed-11ea-9bad-657eed16b965.png">